src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,113 @@
-import React from 'react';
+import React, { FormEvent, useCallback, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api/api';
+import { Movie } from '../../react-app-env';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = React.memo(({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [error, setError] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const onTitleChange = useCallback((
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setQuery(event.target.value);
+    setError('');
+  }, []);
+
+  const handleMovieSearch = useCallback((event: FormEvent) => {
+    event.preventDefault();
+
+    getMovie(query)
+      .then((response: Movie) => {
+        if (response.Error) {
+          setMovie(null);
+          setError(response.Error);
+        } else {
+          setMovie(response);
+          setError('');
+        }
+      });
+  }, []);
+
+  const handleMovieAdd = () => {
+    if (movie) {
+      addMovie(movie);
+    }
```

Comment:
Clear movie after adding it
![image](https://user-images.githubusercontent.com/37161315/179692969-ec7383fe-4813-44ad-b27c-5b963c15c222.png)